libs/community/tests/integration_tests/document_loaders/test_astradb.py

```diff
@@ -0,0 +1,96 @@
+"""
+Test of Astra DB document loader class `AstraDBLoader`
+
+Required to run this test:
+    - a recent `astrapy` Python package available
+    - an Astra DB instance;
+    - the two environment variables set:
+        export ASTRA_DB_API_ENDPOINT="https://<DB-ID>-us-east1.apps.astra.datastax.com"
+        export ASTRA_DB_APPLICATION_TOKEN="AstraCS:........."
+    - optionally this as well (otherwise defaults are used):
+        export ASTRA_DB_KEYSPACE="my_keyspace"
```

Comment:
should we use `namespace` notation (as opposed to _keyspace_) at here, on lines 23, 37, 60 & 89 too just to be consisten everywhere?

Reply:
The name `ASTRA_DB_KEYSPACE ` has been used everywhere for the env var in astrapy, LangChain and RAGStack. I'm OK to change it but then we should change it everywhere. @hemidactylus ?